### PR TITLE
add HAUS logo and tokenlist json

### PR DESCRIPTION
**Pull request by dekanbro**
**Description**

Adding DAOhaus HAUS token logo and tokenlist json entry

### CODE REVIEW

File: optimism.tokenlist.json
Comment (tremarkley) [inline optimism.tokenlist.json:1]: Tokens should not be added directly to the optimism.tokenlist.json. Please revert the changes to this file and add a data.json file under the data/HAUS directory that follows the format specified in the [README](https://github.com/ethereum-optimism/ethereum-optimism.github.io#create-a-data-file)

**Review by tremarkley — DISMISSED** [review]

**Comment by dekanbro** [comment]
Thank you for pointing me in the right direction. I was looking at old docs https://community.optimism.io/docs/developers/bridge/standard-bridge/#the-optimism-token-list

**Review by jakim929 — APPROVED** [review]
